src/langcheck/plot/_utils.py

```diff
@@ -0,0 +1,33 @@
+import re
+from enum import Enum
+
+from plotly.graph_objects import Figure
+
+
+class Axis(Enum):
+    vertical = 0
+    horizontal = 1
+
+
+def _plot_threshold(fig: Figure, threshold_text: str, direction: Axis):
+    # Analyize threshold condition
+    pattern = r'(<|<=|>|>=|==)\s([0-9]\.[0-9]+)'
```

Comment:
This currently fails in cases where the threshold is not a decimal number, e.g. `(flesch_kincaid_grade_values > 5).scatter()`. Instead of extracting the `operator` and `threshold` from the threshold text, how about getting them directly from the `MetricValueWithThreshold` object?

e.g. `_plot_threshold` could be implemented as
```
def _plot_threshold(fig: Figure, operator: str, threshold: Union[float, int], direction: Axis):
    threshold_text = f'{operator} {threshold}'
    if direction == Axis.horizontal:
        ...
```

and then where `_plot_threshold` gets called, it can look like
```
if isinstance(metric_value, MetricValueWithThreshold):
    _plot_threshold(fig, metric_value.threshold_op, metric_value.threshold, Axis.horizontal)
```

Reply:
Thanks for the fix!